### ISSUE TITLE
Add event-handlers for "onFirstStateReceived" and "onConnect"

### ISSUE THREAD

**Issue by niklaswolf** [gamestdio/colyseus-unity3d#12]
As discussed in the gitter channel, it would be nice to have two new event-handlers for `onFirstStateReceived` (to be able to build the initial game structure more easily) and `onConnect`.

**Comment by endel** [comment]
+1 for `onFirstStateReceived`.

I think `onConnect` can be easily confounded with [`onJoin`](https://github.com/gamestdio/colyseus-unity3d/blob/master/Assets/Colyseus/Room.cs#L29)

**Comment by endel** [comment]
Hey @niklaswolf, I'm finally having some time to update the Unity3d client. It would be convenient to create a new property for this on `RoomUpdateEventArgs`.

What do you think?

```csharp
void OnUpdate (object sender, RoomUpdateEventArgs e)
{
    Debug.Log(e.isFirstState);
}
```

**Comment by niklaswolf** [comment]
Hi @endel,
yes this sounds really good! We just have to document this well, so that everyone knows how to use it :)

**Comment by endel** [comment]
Available on 0.5.0 release! https://github.com/gamestdio/colyseus-unity3d/releases/tag/0.5.0
